<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseForm</class>
 <widget class="QWidget" name="MouseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouse</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>415</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>CONTINUOUSLY ACTIVE SLIDERS</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Control key</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QRadioButton" name="ctrlRadioButton">
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Ctrl</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="shiftRadioButton">
              <property name="toolTip">
               <string>Make Shift key control whether sliders are continuously active</string>
              </property>
              <property name="text">
               <string>Shift</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="altRadioButton">
              <property name="toolTip">
               <string>Make Alt key control whether sliders are continuously active (may not work with X11 window managers)</string>
              </property>
              <property name="text">
               <string>Alt</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Active when</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QRadioButton" name="keyUpRadio">
              <property name="toolTip">
               <string>Make sliders active unless you press the control key</string>
              </property>
              <property name="text">
               <string>Key Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="keyDownRadio">
              <property name="toolTip">
               <string>Make sliders active only when you press the control key</string>
              </property>
              <property name="text">
               <string>Key Down</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="neverRadio">
              <property name="toolTip">
               <string>Make sliders never continuously active</string>
              </property>
              <property name="text">
               <string>Never</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="Line1">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel2">
        <property name="text">
         <string>ASSIGNMENT OF MOUSE BUTTONS
(First = pan, set position, attach to point
Second = movie up, add point
Third = movie down, modify point)</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>First, second, third buttons</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QRadioButton" name="lmrRadioButton">
               <property name="text">
                <string>Left, Middle, Right *</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="mrlRadioButton">
               <property name="text">
                <string>Middle, Right, Left</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rlmRadioButton">
               <property name="text">
                <string>Right, Left, Middle **</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="lrmRadioButton">
               <property name="text">
                <string>Left, Right, Middle</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="mlrRadioButton">
               <property name="text">
                <string>Middle, Left, Right</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rmlRadioButton">
               <property name="text">
                <string>Right, Middle, Left</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="defaultLabel">
            <property name="text">
             <string>* Default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="betterLabel">
            <property name="text">
             <string>** May be better for new users</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="leftLabel">
              <property name="text">
               <string>Pan, mark
Attach</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="rightLabel">
              <property name="text">
               <string>Movie down
Modify point</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>150</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/mouseconfig.png</normaloff>:/images/mouseconfig.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>120</width>
                <height>150</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="middleLabel">
            <property name="text">
             <string>Movie up
Add point</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="swapModvCheckBox">
        <property name="toolTip">
         <string>Use left for rotation and middle for shift in model view window</string>
        </property>
        <property name="text">
         <string>Swap left and middle in Model View</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer22">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>253</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="Spacer10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>swapModvCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MouseForm</receiver>
   <slot>swapToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
